### ISSUE TITLE
Allow for different vaccine for 1st dose and second dose

### ISSUE THREAD

**Issue by blanchardjeremy** [microcovid/microcovid#952]
Seeing a couple of requests in our inbox.

> I did the risk evaluation just now, but I am one of the different doses people. I got AstraZeneca before they stopped it here in Sweden for those under 65 and then got Pfizer. Would be cool if you could have the option to have different doses plugged in. Though perhaps there is insufficient data for different combinations? 
> 
> Fun tool either way. I feel a little better about everything after going through some scenarios. 

And another:

> I'm sure I'm not the first to suggest this, but I think it would help make things more accurate if you allowed for people who did not receive the same vaccine for their first and second dose.
> 
> I, for example, received Pfizer for my first dose and Moderna for my second. Your tool seems to assume that if someone has had 2 doses, those doses were both of the same vaccine.
> 
> I imagine making this change would complicate the math, but it would also make your tool available to more people.

**Comment by claberham** [comment]
Yes, please. I received AZ as my first dose and Moderna as my 2nd. Is my risk level the same as someone who received 2 doses of Moderna? Thanks for this tool!

**Comment by hueykwik** [comment]
In Taiwan, I think most people have gotten one shot of AZ, but the government may be offering mixed vaccinations soon.
